integration-tests/dataformats-json/src/main/java/org/apache/camel/quarkus/component/dataformats/json/JsonDataformatsRoute.java

```diff
@@ -39,14 +41,16 @@
 import org.apache.camel.spi.DataFormat;
 import org.apache.johnzon.mapper.reflection.JohnzonParameterizedType;
 
+@ApplicationScoped
 public class JsonDataformatsRoute extends RouteBuilder {
 
+    @Inject
+    ObjectMapper jacksonObjectMapper;
```

Comment:
Is this field used somewhere?

Reply:
Yes, at line 54 of this file

```
jacksonDummyObjectDataFormat.setObjectMapper(jacksonObjectMapper);
```

I just reuse existing test case/jacksonObjectMapper, hope this is OK.

Reply:
Ah, sorry, I have misread the diff. I thought jacksonObjectMapper is a local var on line 54.